web/.env.example

```
@@ -1,6 +1,7 @@
 RESEND_API_KEY=
 CMUX_FEEDBACK_FROM_EMAIL=
 CMUX_FEEDBACK_RATE_LIMIT_ID=
+CMUX_CLIENT_CONFIG_RATE_LIMIT_ID=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The CMUX_CLIENT_CONFIG_RATE_LIMIT_ID key should go before the CMUX_FEEDBACK_FROM_EMAIL key

(UnorderedKey)